docs/administering/ONBOARDING_A_NEW_FUND.md

```diff
@@ -0,0 +1,101 @@
+
+## Main steps
+
+### Information gathering
+
+To provision an app, we'll need the following information. The DARIA team member requesting the provisioning should provide all this information to you.
+
+* Short abbreviation of the fund (e.g. DCAF) - referred to as `NAME`, or `NAME_LOWERCASED` if it should be lowercase
+* Full name of the fund (e.g. DC Abortion Fund) - referred to as `FULL_NAME`
+* Phone number of the abortion fund (e.g. 202-000-1111) - referred to as `FUND_PHONE`
+* Website of the fund (e.g. dcabortionfund.org) - referred to as `FUND_DOMAIN`
+* Name and email of the fund admins (e.g. Susan Everyteen - susan@example.com) - we create initial admin accounts for these people
+* Possibly, the different lines they want.
+
+### Gathering API tokens and secrets
+
+We also need to have some API tokens and such that we generate or dig up when we provision.
+
+* Set up the subdomain in heroku
+  * Open up the pipeline in Heroku and go to the production instance, then open up Settings, then go to Domains.
+  * Click Add Domain and enter in `NAME_LOWERCASED.dariaservices.com`.
+  * Save the DNS target this generates - you'll need it next in Namecheap.
+
+* Set up the DNS in Namecheap
+  * Open up the Namecheap control panel and head to Advanced DNS.
+  * Add a new CNAME Record, with the host of `NAME_LOWERCASED`, and the DNS target from the step above.
+
+* Configure Google OAuth (for Login with Google):
+  * Go to our [GCP project](https://console.cloud.google.com/apis/credentials?project=daria-services-multitenant)
+  * Open up `DARIA Services Production` from the list of Client IDs to edit
+  * Add a new `Authorized redirect URI` using the pattern, using `NAME_LOWERCASED` as the subdomain, and save.
+
+* Insert the new fund record:
+  * Open up a rails console `heroku run rails c -a (multitenant_app_name)`. You're gonna be in here for a few steps, so don't exit out right away.
+  * Insert the fund record as follows: `Fund.create! name: NAME, subdomain: NAME_LOWERCASED, domain: FUND_DOMAIN, full_name: FULL_NAME, phone: FUND_PHONE`. (For example: `Fund.create! name: 'CAT', subdomain: 'cat', domain: 'www.catfund.org', full_name: 'Cat Fund', phone: '281-330-8004')`
+
+* Set that fund as Active for scoping purposes:
+  * `ActsAsTenant.current_tenant = Fund.find_by(name: NAME)`
+
+* Set up the Lines:
+  * If you didn't get explicit instructions on which lines, that means they just need one, `Main`. Run `Line.create name: 'Main'`.
+  * If you DID get a set of lines to create, for every line they want, run `Line.create name: LINE_NAME`. ex: `Line.create name: 'Cats'`.
+
+* Set up the first users, and also do a little smoke testing while you're at it:
+  * Set yourself up first, using this snippet:
```

Comment:
I'm fine with this for now, but ideally this can just be a script the admins (i.e.) me runs. Maybe something to work on next month...